sdk/storage/Azure.Storage.Blobs.ChangeFeed/README.md

```diff
@@ -0,0 +1,148 @@
+```
+
+### Get events between a start and end time
+```C# Snippet:SampleSnippetsChangeFeed_GetEventsBetweenStartAndEndTime
+// Create the start and end time.  The change feed client will round start time down to
+// the nearest hour, and round endTime up to the next hour if you provide DateTimeOffsets
+// with minutes and seconds.
+DateTimeOffset startTime = new DateTimeOffset(2017, 3, 2, 15, 0, 0, TimeSpan.Zero);
+DateTimeOffset endTime = new DateTimeOffset(2020, 10, 7, 2, 0, 0, TimeSpan.Zero);
+
+// You can also provide just a start or end time.
+await foreach (BlobChangeFeedEvent changeFeedEvent in changeFeedClient.GetChangesAsync(
+    start: startTime,
+    end: endTime))
+{
+    changeFeedEvents.Add(changeFeedEvent);
+}
+```
+
+### Resume with cursor
+```C# Snippet:SampleSnippetsChangeFeed_ResumeWithCursor
+IAsyncEnumerator<Page<BlobChangeFeedEvent>> enumerator = changeFeedClient
+    .GetChangesAsync()
+    .AsPages(pageSizeHint: 10)
+    .GetAsyncEnumerator();
+
+await enumerator.MoveNextAsync();
+
+foreach (BlobChangeFeedEvent changeFeedEvent in enumerator.Current.Values)
+{
+    changeFeedEvents.Add(changeFeedEvent);
+}
+
+// get the change feed cursor.  The cursor is not required to get each page of events,
+// it is intended to be saved and used to resume iterating at a later date.
+string cursor = enumerator.Current.ContinuationToken;
+
+// Resume iterating from the pervious position with the cursor.
+await foreach (BlobChangeFeedEvent changeFeedEvent in changeFeedClient.GetChangesAsync(
+    continuation: cursor))
+{
+    changeFeedEvents.Add(changeFeedEvent);
+}
+```
+
+## Troubleshooting
+All Blob service operations will throw a
+[RequestFailedException][RequestFailedException] on failure with
+helpful [`ErrorCode`s][error_codes].  Many of these errors are recoverable.
+
+## Next steps
+
+Get started with our [Change Feed samples][samples]:
+
+1. [Hello World](samples/Sample01a_HelloWorld.cs): Get changes that have occured in your storage account (or [asynchronously](samples/Sample01b_HelloWorldAsync.cs))
+2. [Auth](samples/Sample02_Auth.cs): Authenticate with connection strings, public access, shared keys, shared access signatures, and Azure Active Directory.
+
+
+## Contributing
+
+See the [Storage CONTRIBUTING.md][storage_contrib] for details on building,
+testing, and contributing to this library.
+
+This project welcomes contributions and suggestions.  Most contributions require
+you to agree to a Contributor License Agreement (CLA) declaring that you have
+the right to, and actually do, grant us the rights to use your contribution. For
+details, visit [cla.microsoft.com][cla].
+
+This project has adopted the [Microsoft Open Source Code of Conduct][coc].
+For more information see the [Code of Conduct FAQ][coc_faq]
+or contact [opencode@microsoft.com][coc_contact] with any
+additional questions or comments.
+
+<!-- LINKS -->
+[source]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Blobs/srcs
```

Comment:
Linked to blob rather than change feed.

Reply:
We will change this once Change Feed merges to master.